supabase/migrations/16_tenants.sql

```diff
@@ -7,8 +7,7 @@ create table tenants (
   tenant                  catalog_tenant unique not null,
 
   tasks_quota             integer        not null default 10,
-  collections_quota       integer        not null default 100,
-  l1_stat_rollup          integer        not null default 0
```

Comment:
This column was added as an initial migration with the `ops-catalog` rollout, but is now not needed.